### ISSUE TITLE
Think about how to frame the datasets list page to discourage use

### ISSUE THREAD

**Issue by keithamoss** [govhackaustralia/govhackaustralia.github.io#21]
e.g. Call it _"The Mega List of Datasets"_ to subtly encourage people not to use it and set expectations (it's just a huge list - use the other pages to navigate datasets.)

**Comment by jesserobertson** [comment]
Agree. I mostly stuck that in so that people could find their way back easily to something specific - it's easier to think "that animal dataset" than remember the ALA.
